devel/fetch_metrics/fetch_metrics.py

```diff
@@ -83,29 +83,40 @@ def init_args():
     return args
 
 def print_per_verbose(lvl: int, msg: str):
-    if args['verbose'] < lvl: return
+    if args and (args['verbose'] < lvl): return
     tabs = max(0, lvl - 1)
     print("\t" * tabs + msg)
 
 def load_sortings() -> List[Dict[str, Any]]:
     hydrated_sortings = kp.load_json(args['sortingsfile'])
-    return hydrated_sortings
+    assert hydrated_sortings is not None
+    return cast(List[Dict[str, Any]], hydrated_sortings)
 
-def compute_quality_metrics(recording: RecordingExtractor, sorting: SortingExtractor) -> str:
+def compute_quality_metrics(recording: se.RecordingExtractor, sorting: se.SortingExtractor) -> str:
+    # import within function in case we don't have spiketoolkit installed outside the container
```

Comment:
import within function in case we don't have spiketoolkit installed outside the container